rnaseq-cgl-pipeline/test.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python2.7
+# John Vivian
+import subprocess
+import unittest
+
+
+class TestRNASeqPipeline(unittest.TestCase):
+
+    def test_docker_call(self):
+        tool = ['quay.io/ucsc_cgl/rnaseq-cgl-pipeline']
+        base = ['docker', 'run']
+        args = ['--star=/foo', '--rsem=/foo', '--kallisto=/foo', '--samples=/foo']
+        sock = ['-v', '/var/run/docker.sock:/var/run/docker.sock']
+        mirror = ['-v', '/foo:/foo']
+        sample = ['-v', '/bar:/samples']
+        inputs = ['-v', '/foobar:/inputs']
+        # Check base call for help menu
+        out = check_docker_output(command=base + tool)
+        self.assertTrue('Please see the complete documentation' in out)
+        self.assertFalse('foo bar' in out)
+        # Check for not enough mirror mounts
+        self.assertTrue('IllegalArgumentException' in check_docker_output(base + sock + tool + args))
```

Comment:
All your checks raise this exception. So asserting that this string is present in the output is no proof that this particular check was tripped.